coverage/.last_run.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Cool. Hoping .gitignore will keep these out of github moving forward. Ask Nick E. for advice if .gitignore continues to refuse to ignore.